woocommerce/woocommerce
# Add isSelectableByUser attribute to product templates

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:








### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the new product editor at /wp-admin/admin.php?page=wc-settings&tab=advanced&section=features
2. Go to Products > Add
3. Click on 'Change product type'
4. Check the templates that are listed
5. Install and enable the following plugin:
[test-product-template.zip](https://github.com/woocommerce/woocommerce/files/14922368/test-product-template.zip)
6. Refresh the product page
7. Check the templates again, you should NOT see a new template on the list, since the one added by the plugin has `isSelectableByUser` equal to false


### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
